docs/source/basic/cheatsheet.rst

```diff
@@ -40,6 +40,7 @@ Define accelerator type (CUDA, OpenMP,etc.)
      .. code-block:: c++
 
 	AccGpuCudaRt,
+	AccGpuHipRt,
```

Comment:
What about SYCL?

Reply:
Sure, tell me which names to put there! I wasn't sure myself.

Reply:
alpaka provides the following concrete SYCL targets:

```c++
AccCpuSycl,
AccFpgaSyclIntel,
AccGpuSyclntel
```

Reply:
Thx!